README.md

```diff
@@ -678,6 +679,55 @@ var Color = {
 
 This section describes code style for [ECMAScript 2015 Language Specification](http://www.ecma-international.org/publications/standards/Ecma-262.htm).
 
+### Strict mode
+
+* [Strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) should be used.
+
+  *Explanation:*
+
+  * It prevents nasty [bugs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Changes_in_strict_mode).
+  * Many useful features of language (e.g. [classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), [let declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let), block scopes) are available only in strict mode.
+  * [ES6 modules](https://hacks.mozilla.org/2015/08/es6-in-depth-modules/) are automatically strict mode code.
```

Comment:
I think you meant to say "write in strict mode, to be prepared for switching to ES6 modules"? If that is so, you should extend the last point, because it doesn't sound like a reason for switching but just a statement.
Maybe something like "Your code will can be transitioned to ES6 modules, because they are executed in strict mode"?